bundles/org.openhab.binding.venstarthermostat/src/main/java/org/openhab/binding/venstarthermostat/internal/dto/VenstarRuntime.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.venstarthermostat.internal.dto;
+
+/**
+ * The {@link VenstarSensor} represents a sensor returned from the REST API.
+ *
+ * @author Matthew Davies - Class added to add more functionality to binding
+ */
+public class VenstarRuntime {
+    int ts;
```

Comment:
OK. I will update it. Why is that? Coding convention because the number will eventually become very big?

Reply:
yes, this will overrun a `int` in something like 2038 , which while a long while off, its still better to have consistency in coding conventions other wise we get a mess of ints and longs all over the place.